Formula/gabo.rb

```diff
@@ -0,0 +1,19 @@
+class Gabo < Formula
+  desc "Generates GitHub Actions boilerplate"
+  homepage "https://github.com/ashishb/gabo"
+  url "https://github.com/ashishb/gabo/archive/refs/tags/v0.4.6.tar.gz"
+  sha256 "3a0ad4eaa28ba7c5801ffa0e5a3b2fade09ef597281c2d58451792a75b26df52"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build_prod"
+    bin.install "src/gabo/bin/gabo" => "gabo"
+  end
+
+  test do
+    system "#{bin}/#{name}", "--help"
+    system "#{bin}/#{name}", "--version"
```

Comment:
need better test than just version test

---

We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I added tests to the codebase.
I am running them now instead.
See https://github.com/Homebrew/homebrew-core/pull/140306